addons/portal/static/src/chatter/core/portal_chatter.xml

```diff
@@ -4,7 +4,7 @@
 <t t-name="portal.Chatter">
     <div t-if="state.thread" class="o-mail-Chatter w-100 h-100 flex-grow-1 d-flex pt-2" t-att-class="{ 'row':props.twoColumns, 'flex-column':!props.twoColumns }" t-on-scroll="onScrollDebounced" t-ref="root">
         <div t-if="props.composer" class="o-mail-Chatter-top position-sticky top-0" t-att-class="{ 'col-lg-6':props.twoColumns, 'bg-view':env.inFrontendPortalChatter }" t-att-style="(!props.twoColumns and env.inFrontendPortalChatter and !env.projectSharingId) and 'top: -1px !important; margin-top:-15px; padding-top: 20px'" t-ref="top">
-            <Composer composer="state.thread.composer" autofocus="env.inFrontendPortalChatter ? false : true" mode="'extended'" onPostCallback.bind="onPostCallback" dropzoneRef="rootRef" t-key="props.threadId"/>
+            <Composer composer="state.thread.composer" mode="'extended'" onPostCallback.bind="onPostCallback" dropzoneRef="rootRef" t-key="props.threadId" type="'message'"/>
```

Comment:
This commit also set the composer type to message for portal chatter composer
to make sure the composer placeholder in the project sharing is the same as the
backend when sending a message.

This might be a separate fix since this is something else as what you have in the commit message/title

Reply:
Yes, it could be. But no harm in fixing it here, I think? If it's confusing, I can make a separate PR.

Reply:
And also as a result of this fix (removing `inFrontendPortalChatter`), the composer placeholder will change. So maybe it's better to keep it here instead of a separate fix. I'll explain more about what happens here in the commit message.